code/datums/vote/vote.dm

```diff
@@ -11,17 +11,21 @@
 
 	var/start_time
 	var/time_remaining
+	var/time_set
 	var/status = VOTE_STATUS_PREVOTE
 
 	var/list/result                // The results; format is list(choice = votes).
-	var/results_length = 3         // How many choices to show in the result. Setting to -1 will show all choices.
+	var/results_length = 3         // How many choices to show in the result. Setting to INFINITY will show all choices.
```

Comment:
Is this an actual change? I don't see the relevant code anywhere.

Reply:
This is info update because -1 doesn't work iirc